it/uk/gov/hmrc/minorentityidentificationfrontend/api/controllers/JourneyControllerISpec.scala

```diff
@@ -84,6 +84,7 @@ class JourneyControllerISpec extends ComponentSpecHelper with JourneyStub with A
         stubRetrieveUtr(testJourneyId)(OK, testUtrJson)
```

Comment:
Yeah agreed it's a bit of a weird one because it doesn't take into account which business entity it is or which path their own. The current retrieve data method isn't the most efficient anyway so it might be worth updating in the future